### PR TITLE
Added langauge support for indian languages

### PR DESCRIPTION
**Pull request by amansrivastava17**
Added langauge support for indian languages

### CODE REVIEW

File: ner_v1/detectors/textual/text/text_detection.py [hunk 1]
```diff
@@ -51,7 +51,19 @@ def __init__(self, entity_name=None, source_language_script=lang_constant.ENGLIS
...
+            lang_constant.GUJARATI_LANG,
+            lang_constant.BENGALI_LANG,
+            lang_constant.MARATHI_LANG,
+            lang_constant.TELEGU_LANG,
+            lang_constant.TAMIL_LANG,
+            lang_constant.URDU_LANG,
+            lang_constant.KANNADA_LANG,
+            lang_constant.ORIYA_LANG,
+            lang_constant.MALAYALAM_LANG,
+            lang_constant.PUNJABI_LANG # Added temporarily till text detection is ported to v2 api
```
Comment (srajdev) [inline ner_v1/detectors/textual/text/text_detection.py]: E261 at least two spaces before inline comment

**Comment by srajdev** [comment]
:+1: No lint errors found.

**Comment by haptik-deployment** [comment]
UNIT TESTS FAILED... Please resolve issues

**Comment by haptik-deployment** [comment]
UNIT TESTS HAVE PASSED... Good To Merge

**Comment by amansrivastava17** [comment]
retest this please

**Pull request merged**